tests/yam/validate/validate_desktop.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+
+# Summary: Validate agama GNOME Desktop Environment (Wayland)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    zypper_call('se -i pattern gnome');
```

Comment:
we already have a test module for checking patterns, let's use that one.

Reply:
removed this module and added  `validate_installed_patterns` and `validate_packages` in schedule file and rerun all VRs.

Reply:
All VRs passed with `validate_installed_patterns` and `validate_packages`.